crates/hir_expand/src/name.rs

```diff
@@ -43,8 +43,8 @@ impl Name {
     }
 
     /// Shortcut to create inline plain text name
-    const fn new_inline_ascii(text: &[u8]) -> Name {
-        Name::new_text(SmolStr::new_inline_from_ascii(text.len(), text))
+    const fn new_inline_ascii(text: &str) -> Name {
```

Comment:
Should this method be renamed?

Reply:
Good point, I guess it does make sense. I've renamed it `new_inline()` (see 7b508f2265899d42ea4d7314b606e5cc7837d81f). Not rebased yet to ease the review but I'm not sure at which point of the review process I need to rebase.

Reply:
You can rebase, it's a small change :-).

Reply:
Indeed :stuck_out_tongue: I did rebase this time.